src/Components/test/testassets/BasicTestApp/EventCallbackTest/EventCallbackCases.cshtml

```diff
@@ -0,0 +1,36 @@
+@*
+    Test cases for using EventCallback with various delegate scenarios that used to be troublesome.
+
+    Currently these cases are **VERBOSE** because we haven't yet landed the compiler support for EventCallback.
+    This will be cleaned up soon, and all of the explicit calls to EventCallback.Factory will go away.
```

Comment:
Right. So before you complain, I already explained what's going on 😆. Once the compiler work is done all of the rest of this code in the file will become idiomatic.

Reply:
Understood!